lib/smart_proxy_openscap/openscap_lib.rb

```diff
@@ -120,6 +122,11 @@ def self.save_or_serve_scap_file(policy_id, policy_scap_file)
     end
   end
 
+  def self.fullpath(p = nil)
```

Comment:
few comments

`self.fullpath(p = Proxy::OpenSCAP::Plugin.settings.contentdir)`
to avoid line below?

ideally name the variable like path, I prefer one letter variables on single line blocks

I don't see any invocation that would need the p parameter to be optional, if you want to provide this method for the whole plugin then I'd argue that the default value does not make sense and fullpath should require argument explicitly. If you meant to have fullpath method just for get_policy_content then I don't think you need to pass the argument.